compile failed on ios platform

i aleady modify my podfile 

post_install do |installer|
  installer.pods_project.targets.each do |target|
    target.build_configurations.each do |config|
      config.build_settings['ENABLE_BITCODE'] = 'NO'
      config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = '11.0'

    end
  end
end


but still compile failed on ios . 






Xcode's output:
↳
    /Users/njvivo/Desktop/flutter/.pub-cache/hosted/pub.dartlang.org/webview_flutter-0.3.22+1/ios/Classes/FlutterWebView.m:367:23: warning: 'mediaPlaybackRequiresUserAction' is deprecated: first deprecated in iOS 9.0 [-Wdeprecated-declarations]
            configuration.mediaPlaybackRequiresUserAction = true;
                          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                          mediaTypesRequiringUserActionForPlayback
    In module 'WebKit' imported from /Users/njvivo/Desktop/flutter/.pub-cache/hosted/pub.dartlang.org/webview_flutter-0.3.22+1/ios/Classes/FlutterWebView.h:6:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/WebKit.framework/Headers/WKWebViewConfiguration.h:208:28: note: 'mediaPlaybackRequiresUserAction' has
    been explicitly marked deprecated here
    @property (nonatomic) BOOL mediaPlaybackRequiresUserAction API_DEPRECATED_WITH_REPLACEMENT("mediaTypesRequiringUserActionForPlayback", ios(8.0, 9.0));
                               ^
    /Users/njvivo/Desktop/flutter/.pub-cache/hosted/pub.dartlang.org/webview_flutter-0.3.22+1/ios/Classes/FlutterWebView.m:374:23: warning: 'mediaPlaybackRequiresUserAction' is deprecated: first deprecated in iOS 9.0 [-Wdeprecated-declarations]
            configuration.mediaPlaybackRequiresUserAction = false;
                          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                          mediaTypesRequiringUserActionForPlayback
    In module 'WebKit' imported from /Users/njvivo/Desktop/flutter/.pub-cache/hosted/pub.dartlang.org/webview_flutter-0.3.22+1/ios/Classes/FlutterWebView.h:6:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/WebKit.framework/Headers/WKWebViewConfiguration.h:208:28: note: 'mediaPlaybackRequiresUserAction' has
    been explicitly marked deprecated here
    @property (nonatomic) BOOL mediaPlaybackRequiresUserAction API_DEPRECATED_WITH_REPLACEMENT("mediaTypesRequiringUserActionForPlayback", ios(8.0, 9.0));
                               ^
    2 warnings generated.
    ld: warning: Could not find or use auto-linked library 'swiftCore'
    ld: warning: Could not find or use auto-linked library 'swiftQuartzCore'
    ld: warning: Could not find or use auto-linked library 'swiftCoreGraphics'
    ld: warning: Could not find or use auto-linked library 'swiftFoundation'
    ld: warning: Could not find or use auto-linked library 'swiftMetal'
    ld: warning: Could not find or use auto-linked library 'swiftDarwin'
    ld: warning: Could not find or use auto-linked library 'swiftUIKit'
    ld: warning: Could not find or use auto-linked library 'swiftObjectiveC'
    ld: warning: Could not find or use auto-linked library 'swiftCoreFoundation'
    ld: warning: Could not find or use auto-linked library 'swiftCompatibility50'
    ld: warning: Could not find or use auto-linked library 'swiftCoreImage'
    ld: warning: Could not find or use auto-linked library 'swiftCompatibilityDynamicReplacements'
    ld: warning: Could not find or use auto-linked library 'swiftDispatch'
    ld: warning: Could not find or use auto-linked library 'swiftCoreMedia'
    ld: warning: Could not find or use auto-linked library 'swiftCoreAudio'
    ld: warning: Could not find or use auto-linked library 'swiftSwiftOnoneSupport'
    Undefined symbols for architecture x86_64:
      "base conformance descriptor for Swift.Hashable: Swift.Equatable", referenced from:
          protocol conformance descriptor for __C.NSHTTPCookiePropertyKey : Swift.Hashable in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "method descriptor for Swift.Hashable._rawHashValue(seed: Swift.Int) -> Swift.Int", referenced from:
          protocol conformance descriptor for __C.NSHTTPCookiePropertyKey : Swift.Hashable in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "protocol descriptor for Swift._ObjectiveCBridgeable", referenced from:
          protocol conformance descriptor for __C.NSHTTPCookiePropertyKey : Swift._ObjectiveCBridgeable in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "method descriptor for static Swift._ObjectiveCBridgeable._conditionallyBridgeFromObjectiveC(_: A._ObjectiveCType, result: inout A?) -> Swift.Bool", referenced from:
          protocol conformance descriptor for __C.NSHTTPCookiePropertyKey : Swift._ObjectiveCBridgeable in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "protocol descriptor for Swift.Equatable", referenced from:
          protocol conformance descriptor for __C.NSHTTPCookiePropertyKey : Swift.Equatable in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "base conformance descriptor for Swift._SwiftNewtypeWrapper: Swift.RawRepresentable", referenced from:
          protocol conformance descriptor for __C.NSHTTPCookiePropertyKey : Swift._SwiftNewtypeWrapper in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "protocol descriptor for Swift.RawRepresentable", referenced from:
          protocol conformance descriptor for __C.NSHTTPCookiePropertyKey : Swift.RawRepresentable in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "protocol descriptor for Swift._SwiftNewtypeWrapper", referenced from:
          protocol conformance descriptor for __C.NSHTTPCookiePropertyKey : Swift._SwiftNewtypeWrapper in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "method descriptor for Swift.RawRepresentable.init(rawValue: A.RawValue) -> A?", referenced from:
          protocol conformance descriptor for __C.NSHTTPCookiePropertyKey : Swift.RawRepresentable in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "method descriptor for Swift._ObjectiveCBridgeable._bridgeToObjectiveC() -> A._ObjectiveCType", referenced from:
          protocol conformance descriptor for __C.NSHTTPCookiePropertyKey : Swift._ObjectiveCBridgeable in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "method descriptor for Swift.RawRepresentable.rawValue.getter : A.RawValue", referenced from:
          protocol conformance descriptor for __C.NSHTTPCookiePropertyKey : Swift.RawRepresentable in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "nominal type descriptor for Swift.IndexingIterator", referenced from:
          _symbolic _____ySaySo12NSDictionaryCGG s16IndexingIteratorV in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          _symbolic _____ySaySo12NSHTTPCookieCGG s16IndexingIteratorV in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "method descriptor for Swift._HasCustomAnyHashableRepresentation._toCustomAnyHashable() -> Swift.AnyHashable?", referenced from:
          protocol conformance descriptor for __C.NSHTTPCookiePropertyKey : Swift._HasCustomAnyHashableRepresentation in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "protocol conformance descriptor for Swift.String : Swift._ObjectiveCBridgeable in Foundation", referenced from:
          lazy protocol witness table accessor for type Swift.String and conformance Swift.String : Swift._ObjectiveCBridgeable in Foundation in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "(extension in Swift):Swift._SwiftNewtypeWrapper< where A: Swift.Hashable, A.Swift.RawRepresentable.RawValue: Swift.Hashable>._toCustomAnyHashable() -> Swift.AnyHashable?", referenced from:
          protocol witness for Swift._HasCustomAnyHashableRepresentation._toCustomAnyHashable() -> Swift.AnyHashable? in conformance __C.NSHTTPCookiePropertyKey : Swift._HasCustomAnyHashableRepresentation in __C_Synthesized in
          libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "(extension in Swift):Swift._SwiftNewtypeWrapper< where A: Swift.Hashable, A.Swift.RawRepresentable.RawValue: Swift.Hashable>._rawHashValue(seed: Swift.Int) -> Swift.Int", referenced from:
          protocol witness for Swift.Hashable._rawHashValue(seed: Swift.Int) -> Swift.Int in conformance __C.NSHTTPCookiePropertyKey : Swift.Hashable in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "method descriptor for Swift.Hashable.hash(into: inout Swift.Hasher) -> ()", referenced from:
          protocol conformance descriptor for __C.NSHTTPCookiePropertyKey : Swift.Hashable in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "(extension in Swift):Swift._SwiftNewtypeWrapper< where A: Swift.Hashable, A.Swift.RawRepresentable.RawValue: Swift.Hashable>.hash(into: inout Swift.Hasher) -> ()", referenced from:
          protocol witness for Swift.Hashable.hash(into: inout Swift.Hasher) -> () in conformance __C.NSHTTPCookiePropertyKey : Swift.Hashable in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "protocol witness table for Swift.String : Swift.Hashable in Swift", referenced from:
          protocol witness for Swift.Hashable.hashValue.getter : Swift.Int in conformance __C.NSHTTPCookiePropertyKey : Swift.Hashable in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          protocol witness for Swift.Hashable.hash(into: inout Swift.Hasher) -> () in conformance __C.NSHTTPCookiePropertyKey : Swift.Hashable in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          protocol witness for Swift.Hashable._rawHashValue(seed: Swift.Int) -> Swift.Int in conformance __C.NSHTTPCookiePropertyKey : Swift.Hashable in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          protocol witness for Swift._HasCustomAnyHashableRepresentation._toCustomAnyHashable() -> Swift.AnyHashable? in conformance __C.NSHTTPCookiePropertyKey : Swift._HasCustomAnyHashableRepresentation in __C_Synthesized in
          libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "static (extension in Swift):Swift._SwiftNewtypeWrapper< where A.RawValue: Swift._ObjectiveCBridgeable>._conditionallyBridgeFromObjectiveC(_: A.RawValue._ObjectiveCType, result: inout A?) -> Swift.Bool", referenced from:
          protocol witness for static Swift._ObjectiveCBridgeable._conditionallyBridgeFromObjectiveC(_: A._ObjectiveCType, result: inout A?) -> Swift.Bool in conformance __C.NSHTTPCookiePropertyKey : Swift._ObjectiveCBridgeable in __C_Synthesized
          in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "static (extension in Swift):Swift._SwiftNewtypeWrapper< where A.RawValue: Swift._ObjectiveCBridgeable>._forceBridgeFromObjectiveC(_: A.RawValue._ObjectiveCType, result: inout A?) -> ()", referenced from:
          protocol witness for static Swift._ObjectiveCBridgeable._forceBridgeFromObjectiveC(_: A._ObjectiveCType, result: inout A?) -> () in conformance __C.NSHTTPCookiePropertyKey : Swift._ObjectiveCBridgeable in __C_Synthesized in
          libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "_swift_getForeignTypeMetadata", referenced from:
          type metadata accessor for __C.NSHTTPCookiePropertyKey in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "static (extension in Swift):Swift._SwiftNewtypeWrapper< where A.RawValue: Swift._ObjectiveCBridgeable>._unconditionallyBridgeFromObjectiveC(A.RawValue._ObjectiveCType?) -> A", referenced from:
          protocol witness for static Swift._ObjectiveCBridgeable._unconditionallyBridgeFromObjectiveC(A._ObjectiveCType?) -> A in conformance __C.NSHTTPCookiePropertyKey : Swift._ObjectiveCBridgeable in __C_Synthesized in
          libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "(extension in Foundation):Swift.Bool._bridgeToObjectiveC() -> __C.NSNumber", referenced from:
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin._cookieToDictionary(cookie: __C.NSHTTPCookie) -> __C.NSDictionary in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "associated type descriptor for Swift.RawRepresentable.RawValue", referenced from:
          protocol conformance descriptor for __C.NSHTTPCookiePropertyKey : Swift.RawRepresentable in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "static Foundation.Date._unconditionallyBridgeFromObjectiveC(__C.NSDate?) -> Foundation.Date", referenced from:
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin._cookieToDictionary(cookie: __C.NSHTTPCookie) -> __C.NSDictionary in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "method descriptor for Swift.Hashable.hashValue.getter : Swift.Int", referenced from:
          protocol conformance descriptor for __C.NSHTTPCookiePropertyKey : Swift.Hashable in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "protocol conformance descriptor for [A] : Swift.Collection in Swift", referenced from:
          lazy protocol witness table accessor for type [__C.NSDictionary] and conformance [A] : Swift.Collection in Swift in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          lazy protocol witness table accessor for type [__C.NSHTTPCookie] and conformance [A] : Swift.Collection in Swift in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "(extension in Foundation):Swift.Dictionary._bridgeToObjectiveC() -> __C.NSDictionary", referenced from:
          @nonobjc __C.NSHTTPCookie.init(properties: [__C.NSHTTPCookiePropertyKey : Any]) -> __C.NSHTTPCookie? in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "_swift_getWitnessTable", referenced from:
          lazy protocol witness table accessor for type [__C.NSDictionary] and conformance [A] : Swift.Collection in Swift in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          lazy protocol witness table accessor for type __C.NSHTTPCookiePropertyKey and conformance __C.NSHTTPCookiePropertyKey : Swift.Hashable in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          lazy protocol witness table accessor for type [__C.NSHTTPCookie] and conformance [A] : Swift.Collection in Swift in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          lazy protocol witness table accessor for type Swift.String and conformance Swift.String : Swift.StringProtocol in Swift in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          lazy protocol witness table accessor for type __C.NSHTTPCookiePropertyKey and conformance __C.NSHTTPCookiePropertyKey : Swift.RawRepresentable in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          lazy protocol witness table accessor for type __C.NSHTTPCookiePropertyKey and conformance __C.NSHTTPCookiePropertyKey : Swift._HasCustomAnyHashableRepresentation in __C_Synthesized in
          libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          lazy protocol witness table accessor for type __C.NSHTTPCookiePropertyKey and conformance __C.NSHTTPCookiePropertyKey : Swift.Equatable in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          ...
      "type metadata accessor for Swift.Optional", referenced from:
          closure #1 ([__C.NSHTTPCookie]) -> () in static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.getCookies(url: Swift.String?, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin._cookieToDictionary(cookie: __C.NSHTTPCookie) -> __C.NSDictionary in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          outlined init with copy of Foundation.Date? in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          outlined init with take of Foundation.Date? in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "Swift.== infix<A where A: Swift.RawRepresentable, A.RawValue: Swift.Equatable>(A, A) -> Swift.Bool", referenced from:
          protocol witness for static Swift.Equatable.== infix(A, A) -> Swift.Bool in conformance __C.NSHTTPCookiePropertyKey : Swift.Equatable in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "Swift._allocateUninitializedArray<A>(Builtin.Word) -> ([A], Builtin.RawPointer)", referenced from:
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.(_setCookie in _E35A6B25B6E5F14C2C28AC213C9247D2)(cookie: __C.NSDictionary, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "Foundation.URL.init(string: __shared Swift.String) -> Foundation.URL?", referenced from:
          closure #1 ([__C.NSHTTPCookie]) -> () in static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.getCookies(url: Swift.String?, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "method descriptor for static Swift._ObjectiveCBridgeable._unconditionallyBridgeFromObjectiveC(A._ObjectiveCType?) -> A", referenced from:
          protocol conformance descriptor for __C.NSHTTPCookiePropertyKey : Swift._ObjectiveCBridgeable in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "type metadata accessor for Foundation.Date", referenced from:
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.(_setCookie in _E35A6B25B6E5F14C2C28AC213C9247D2)(cookie: __C.NSDictionary, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin._cookieToDictionary(cookie: __C.NSHTTPCookie) -> __C.NSDictionary in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          outlined init with copy of Foundation.Date? in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          outlined destroy of Foundation.Date? in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          outlined init with take of Foundation.Date? in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "type metadata accessor for Swift.Array", referenced from:
          type metadata accessor for [__C.NSDictionary] in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          type metadata accessor for [__C.NSHTTPCookie] in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "Foundation.Date.timeIntervalSince1970.getter : Swift.Double", referenced from:
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin._cookieToDictionary(cookie: __C.NSHTTPCookie) -> __C.NSDictionary in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "static (extension in Foundation):Swift.Array._unconditionallyBridgeFromObjectiveC(__C.NSArray?) -> [A]", referenced from:
          reabstraction thunk helper from @escaping @callee_guaranteed (@guaranteed [__C.NSHTTPCookie]) -> () to @escaping @callee_unowned @convention(block) (@unowned __C.NSArray) -> () in
          libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "_swift_endAccess", referenced from:
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.httpCookieStore.getter : __C.WKHTTPCookieStore? in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.httpCookieStore.setter : __C.WKHTTPCookieStore? in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.httpCookieStore.modify : __C.WKHTTPCookieStore? with unmangled suffix ".resume.0" in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.register(with: __C.FlutterPluginRegistrar) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          webview_cookie_manager.SwiftWebviewCookieManagerPlugin.handle(_: __C.FlutterMethodCall, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.clearCookies(result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.getCookies(url: Swift.String?, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          ...
      "Swift.Dictionary.init(dictionaryLiteral: (A, B)...) -> [A : B]", referenced from:
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.(_setCookie in _E35A6B25B6E5F14C2C28AC213C9247D2)(cookie: __C.NSDictionary, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "Foundation.URL.host.getter : Swift.String?", referenced from:
          closure #1 ([__C.NSHTTPCookie]) -> () in static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.getCookies(url: Swift.String?, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "_swift_deallocObject", referenced from:
          l_objectdestroy in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          l_objectdestroy.2 in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          l_objectdestroy.8 in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          l_objectdestroy.12 in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "(extension in Foundation):Swift.Int._bridgeToObjectiveC() -> __C.NSNumber", referenced from:
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin._cookieToDictionary(cookie: __C.NSHTTPCookie) -> __C.NSDictionary in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "(extension in Foundation):Swift.StringProtocol.contains<A where A1: Swift.StringProtocol>(A1) -> Swift.Bool", referenced from:
          closure #1 ([__C.NSHTTPCookie]) -> () in static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.getCookies(url: Swift.String?, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "protocol witness table for Swift.String : Swift.Equatable in Swift", referenced from:
          protocol witness for static Swift.Equatable.== infix(A, A) -> Swift.Bool in conformance __C.NSHTTPCookiePropertyKey : Swift.Equatable in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "type metadata for Swift.Bool", referenced from:
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.(_setCookie in _E35A6B25B6E5F14C2C28AC213C9247D2)(cookie: __C.NSDictionary, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          closure #1 ([__C.NSHTTPCookie]) -> () in static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.clearCookies(result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          closure #1 () -> () in static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.(_setCookie in _E35A6B25B6E5F14C2C28AC213C9247D2)(cookie: __C.NSDictionary, result: (Any?) -> ()) -> () in
          libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "(extension in Swift):Swift.Collection< where A.Iterator == Swift.IndexingIterator<A>>.makeIterator() -> Swift.IndexingIterator<A>", referenced from:
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.setCookies(cookies: [__C.NSDictionary], result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          closure #1 ([__C.NSHTTPCookie]) -> () in static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.clearCookies(result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          closure #1 ([__C.NSHTTPCookie]) -> () in static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.getCookies(url: Swift.String?, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "Swift.IndexingIterator.next() -> A.Element?", referenced from:
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.setCookies(cookies: [__C.NSDictionary], result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          closure #1 ([__C.NSHTTPCookie]) -> () in static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.clearCookies(result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          closure #1 ([__C.NSHTTPCookie]) -> () in static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.getCookies(url: Swift.String?, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "Swift.Double.isFinite.getter : Swift.Bool", referenced from:
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin._cookieToDictionary(cookie: __C.NSHTTPCookie) -> __C.NSDictionary in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "method descriptor for static Swift.Equatable.== infix(A, A) -> Swift.Bool", referenced from:
          protocol conformance descriptor for __C.NSHTTPCookiePropertyKey : Swift.Equatable in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "_swift_getTypeByMangledNameInContext", referenced from:
          ___swift_instantiateConcreteTypeFromMangledName in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "static (extension in Foundation):Swift.String._unconditionallyBridgeFromObjectiveC(__C.NSString?) -> Swift.String", referenced from:
          webview_cookie_manager.SwiftWebviewCookieManagerPlugin.handle(_: __C.FlutterMethodCall, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          closure #1 ([__C.NSHTTPCookie]) -> () in static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.getCookies(url: Swift.String?, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin._cookieToDictionary(cookie: __C.NSHTTPCookie) -> __C.NSDictionary in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          __C.NSHTTPCookiePropertyKey.rawValue.getter : Swift.String in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "_swift_allocBox", referenced from:
          ___swift_allocate_boxed_opaque_existential_0 in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "_swift_bridgeObjectRelease", referenced from:
          @nonobjc __C.FlutterMethodChannel.__allocating_init(name: Swift.String, binaryMessenger: __C.FlutterBinaryMessenger) -> __C.FlutterMethodChannel in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          webview_cookie_manager.SwiftWebviewCookieManagerPlugin.handle(_: __C.FlutterMethodCall, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          outlined consume of Swift.String? in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.(_setCookie in _E35A6B25B6E5F14C2C28AC213C9247D2)(cookie: __C.NSDictionary, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          reabstraction thunk helper from @escaping @callee_guaranteed (@guaranteed [__C.NSHTTPCookie]) -> () to @escaping @callee_unowned @convention(block) (@unowned __C.NSArray) -> () in
          libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          @nonobjc __C.NSHTTPCookie.init(properties: [__C.NSHTTPCookiePropertyKey : Any]) -> __C.NSHTTPCookie? in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          closure #1 ([__C.NSHTTPCookie]) -> () in static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.getCookies(url: Swift.String?, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          ...
      "protocol conformance descriptor for Swift.String : Swift.StringProtocol in Swift", referenced from:
          lazy protocol witness table accessor for type Swift.String and conformance Swift.String : Swift.StringProtocol in Swift in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "type metadata for Swift.String", referenced from:
          webview_cookie_manager.SwiftWebviewCookieManagerPlugin.handle(_: __C.FlutterMethodCall, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.(_setCookie in _E35A6B25B6E5F14C2C28AC213C9247D2)(cookie: __C.NSDictionary, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          closure #1 ([__C.NSHTTPCookie]) -> () in static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.getCookies(url: Swift.String?, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          lazy protocol witness table accessor for type Swift.String and conformance Swift.String : Swift.StringProtocol in Swift in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          lazy protocol witness table accessor for type Swift.String and conformance Swift.String : Swift._ObjectiveCBridgeable in Foundation in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "__swift_FORCE_LOAD_$_swiftCompatibilityDynamicReplacements", referenced from:
          __swift_FORCE_LOAD_$_swiftCompatibilityDynamicReplacements_$_webview_cookie_manager in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
         (maybe you meant: __swift_FORCE_LOAD_$_swiftCompatibilityDynamicReplacements_$_webview_cookie_manager)
      "__swift_FORCE_LOAD_$_swiftCompatibility50", referenced from:
          __swift_FORCE_LOAD_$_swiftCompatibility50_$_webview_cookie_manager in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
         (maybe you meant: __swift_FORCE_LOAD_$_swiftCompatibility50_$_webview_cookie_manager)
      "(extension in Swift):Swift._SwiftNewtypeWrapper< where A.RawValue: Swift._ObjectiveCBridgeable>._bridgeToObjectiveC() -> A.RawValue._ObjectiveCType", referenced from:
          protocol witness for Swift._ObjectiveCBridgeable._bridgeToObjectiveC() -> A._ObjectiveCType in conformance __C.NSHTTPCookiePropertyKey : Swift._ObjectiveCBridgeable in __C_Synthesized in
          libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "_swift_bridgeObjectRetain", referenced from:
          webview_cookie_manager.SwiftWebviewCookieManagerPlugin.handle(_: __C.FlutterMethodCall, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.setCookies(cookies: [__C.NSDictionary], result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.(_setCookie in _E35A6B25B6E5F14C2C28AC213C9247D2)(cookie: __C.NSDictionary, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          closure #1 ([__C.NSHTTPCookie]) -> () in static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.clearCookies(result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          closure #1 ([__C.NSHTTPCookie]) -> () in static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.getCookies(url: Swift.String?, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          outlined copy of Swift.String? in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          outlined init with copy of Swift.String? in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          ...
      "_swift_dynamicCast", referenced from:
          webview_cookie_manager.SwiftWebviewCookieManagerPlugin.handle(_: __C.FlutterMethodCall, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.(_setCookie in _E35A6B25B6E5F14C2C28AC213C9247D2)(cookie: __C.NSDictionary, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "_swift_retain", referenced from:
          webview_cookie_manager.SwiftWebviewCookieManagerPlugin.handle(_: __C.FlutterMethodCall, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.clearCookies(result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.getCookies(url: Swift.String?, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.(_setCookie in _E35A6B25B6E5F14C2C28AC213C9247D2)(cookie: __C.NSDictionary, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          closure #1 ([__C.NSHTTPCookie]) -> () in static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.clearCookies(result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          reabstraction thunk helper from @escaping @callee_guaranteed (@guaranteed [__C.NSHTTPCookie]) -> () to @escaping @callee_unowned @convention(block) (@unowned __C.NSArray) -> () in
          libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          _block_copy_helper in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          ...
      "_swift_once", referenced from:
          webview_cookie_manager.SwiftWebviewCookieManagerPlugin.httpCookieStore.unsafeMutableAddressor : __C.WKHTTPCookieStore? in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "_swift_unknownObjectRetain", referenced from:
          @objc static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.register(with: __C.FlutterPluginRegistrar) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "protocol descriptor for Swift.Hashable", referenced from:
          protocol conformance descriptor for __C.NSHTTPCookiePropertyKey : Swift.Hashable in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "(extension in Foundation):Swift.String._bridgeToObjectiveC() -> __C.NSString", referenced from:
          @nonobjc __C.FlutterMethodChannel.__allocating_init(name: Swift.String, binaryMessenger: __C.FlutterBinaryMessenger) -> __C.FlutterMethodChannel in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.(_setCookie in _E35A6B25B6E5F14C2C28AC213C9247D2)(cookie: __C.NSDictionary, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin._cookieToDictionary(cookie: __C.NSHTTPCookie) -> __C.NSDictionary in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          __C.NSHTTPCookiePropertyKey.init(rawValue: Swift.String) -> __C.NSHTTPCookiePropertyKey in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "_swift_getObjCClassFromMetadata", referenced from:
          @nonobjc __C.FlutterMethodChannel.__allocating_init(name: Swift.String, binaryMessenger: __C.FlutterBinaryMessenger) -> __C.FlutterMethodChannel in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          __C.NSHTTPCookie.__allocating_init(properties: [__C.NSHTTPCookiePropertyKey : Any]) -> __C.NSHTTPCookie? in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          __C.NSMutableArray.__allocating_init() -> __C.NSMutableArray in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          __C.NSMutableDictionary.__allocating_init() -> __C.NSMutableDictionary in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "associated type descriptor for Swift._ObjectiveCBridgeable._ObjectiveCType", referenced from:
          protocol conformance descriptor for __C.NSHTTPCookiePropertyKey : Swift._ObjectiveCBridgeable in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "Foundation.Date.init(timeIntervalSince1970: Swift.Double) -> Foundation.Date", referenced from:
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.(_setCookie in _E35A6B25B6E5F14C2C28AC213C9247D2)(cookie: __C.NSDictionary, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "_swift_getObjCClassMetadata", referenced from:
          type metadata accessor for __C.FlutterMethodChannel in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          @objc static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.register(with: __C.FlutterPluginRegistrar) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          type metadata accessor for __C.NSObject in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          type metadata accessor for __C.NSDictionary in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          type metadata accessor for __C.NSMutableArray in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          type metadata accessor for __C.NSMutableDictionary in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          type metadata accessor for __C.NSHTTPCookie in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          ...
      "_swift_getInitializedObjCClass", referenced from:
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.register(with: __C.FlutterPluginRegistrar) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          type metadata accessor for __C.FlutterMethodChannel in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          type metadata accessor for webview_cookie_manager.SwiftWebviewCookieManagerPlugin in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          type metadata accessor for __C.NSObject in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          type metadata accessor for __C.NSDictionary in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          type metadata accessor for __C.NSMutableArray in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          type metadata accessor for __C.NSMutableDictionary in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          ...
      "Swift._bridgeAnythingToObjectiveC<A>(A) -> Swift.AnyObject", referenced from:
          webview_cookie_manager.SwiftWebviewCookieManagerPlugin.handle(_: __C.FlutterMethodCall, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          reabstraction thunk helper from @escaping @callee_unowned @convention(block) (@unowned Swift.AnyObject?) -> () to @escaping @callee_guaranteed (@in_guaranteed Any?) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.(_setCookie in _E35A6B25B6E5F14C2C28AC213C9247D2)(cookie: __C.NSDictionary, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "type metadata for Any", referenced from:
          webview_cookie_manager.SwiftWebviewCookieManagerPlugin.handle(_: __C.FlutterMethodCall, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.(_setCookie in _E35A6B25B6E5F14C2C28AC213C9247D2)(cookie: __C.NSDictionary, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          @nonobjc __C.NSHTTPCookie.init(properties: [__C.NSHTTPCookiePropertyKey : Any]) -> __C.NSHTTPCookie? in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "Swift.String.init(_builtinStringLiteral: Builtin.RawPointer, utf8CodeUnitCount: Builtin.Word, isASCII: Builtin.Int1) -> Swift.String", referenced from:
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.register(with: __C.FlutterPluginRegistrar) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          webview_cookie_manager.SwiftWebviewCookieManagerPlugin.handle(_: __C.FlutterMethodCall, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.(_setCookie in _E35A6B25B6E5F14C2C28AC213C9247D2)(cookie: __C.NSDictionary, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin._cookieToDictionary(cookie: __C.NSHTTPCookie) -> __C.NSDictionary in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "method descriptor for static Swift._ObjectiveCBridgeable._forceBridgeFromObjectiveC(_: A._ObjectiveCType, result: inout A?) -> ()", referenced from:
          protocol conformance descriptor for __C.NSHTTPCookiePropertyKey : Swift._ObjectiveCBridgeable in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "Swift.Dictionary.subscript.setter : (A) -> B?", referenced from:
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.(_setCookie in _E35A6B25B6E5F14C2C28AC213C9247D2)(cookie: __C.NSDictionary, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "protocol descriptor for Swift._HasCustomAnyHashableRepresentation", referenced from:
          protocol conformance descriptor for __C.NSHTTPCookiePropertyKey : Swift._HasCustomAnyHashableRepresentation in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "_swift_beginAccess", referenced from:
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.httpCookieStore.getter : __C.WKHTTPCookieStore? in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.httpCookieStore.setter : __C.WKHTTPCookieStore? in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.httpCookieStore.modify : __C.WKHTTPCookieStore? in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.register(with: __C.FlutterPluginRegistrar) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          webview_cookie_manager.SwiftWebviewCookieManagerPlugin.handle(_: __C.FlutterMethodCall, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.clearCookies(result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.getCookies(url: Swift.String?, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          ...
      "_swift_allocObject", referenced from:
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.clearCookies(result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.getCookies(url: Swift.String?, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          @objc webview_cookie_manager.SwiftWebviewCookieManagerPlugin.handle(_: __C.FlutterMethodCall, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.(_setCookie in _E35A6B25B6E5F14C2C28AC213C9247D2)(cookie: __C.NSDictionary, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "type metadata accessor for Foundation.URL", referenced from:
          closure #1 ([__C.NSHTTPCookie]) -> () in static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.getCookies(url: Swift.String?, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "Swift._bridgeAnyObjectToAny(Swift.AnyObject?) -> Any", referenced from:
          webview_cookie_manager.SwiftWebviewCookieManagerPlugin.handle(_: __C.FlutterMethodCall, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.(_setCookie in _E35A6B25B6E5F14C2C28AC213C9247D2)(cookie: __C.NSDictionary, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "(extension in Swift):Swift._SwiftNewtypeWrapper< where A: Swift.Hashable, A.Swift.RawRepresentable.RawValue: Swift.Hashable>.hashValue.getter : Swift.Int", referenced from:
          protocol witness for Swift.Hashable.hashValue.getter : Swift.Int in conformance __C.NSHTTPCookiePropertyKey : Swift.Hashable in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "type metadata for Swift.Double", referenced from:
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.(_setCookie in _E35A6B25B6E5F14C2C28AC213C9247D2)(cookie: __C.NSDictionary, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "base conformance descriptor for Swift._SwiftNewtypeWrapper: Swift._HasCustomAnyHashableRepresentation", referenced from:
          protocol conformance descriptor for __C.NSHTTPCookiePropertyKey : Swift._SwiftNewtypeWrapper in __C_Synthesized in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "_swift_getObjectType", referenced from:
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.register(with: __C.FlutterPluginRegistrar) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "Swift._assertionFailure(_: Swift.StaticString, _: Swift.StaticString, file: Swift.StaticString, line: Swift.UInt, flags: Swift.UInt32) -> Swift.Never", referenced from:
          webview_cookie_manager.SwiftWebviewCookieManagerPlugin.handle(_: __C.FlutterMethodCall, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.clearCookies(result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.getCookies(url: Swift.String?, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.(_setCookie in _E35A6B25B6E5F14C2C28AC213C9247D2)(cookie: __C.NSDictionary, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          closure #1 ([__C.NSHTTPCookie]) -> () in static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.clearCookies(result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          closure #1 ([__C.NSHTTPCookie]) -> () in static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.getCookies(url: Swift.String?, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin._cookieToDictionary(cookie: __C.NSHTTPCookie) -> __C.NSDictionary in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          ...
      "_swift_release", referenced from:
          webview_cookie_manager.SwiftWebviewCookieManagerPlugin.handle(_: __C.FlutterMethodCall, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          ___swift_destroy_boxed_opaque_existential_0 in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.clearCookies(result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.getCookies(url: Swift.String?, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          @objc webview_cookie_manager.SwiftWebviewCookieManagerPlugin.handle(_: __C.FlutterMethodCall, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.(_setCookie in _E35A6B25B6E5F14C2C28AC213C9247D2)(cookie: __C.NSDictionary, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          closure #1 ([__C.NSHTTPCookie]) -> () in static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.clearCookies(result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          ...
      "value witness table for Builtin.UnknownObject", referenced from:
          full type metadata for __C.NSHTTPCookiePropertyKey in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          full type metadata for webview_cookie_manager.SwiftWebviewCookieManagerPlugin in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
      "_swift_unknownObjectRelease", referenced from:
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.register(with: __C.FlutterPluginRegistrar) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          @nonobjc __C.FlutterMethodChannel.__allocating_init(name: Swift.String, binaryMessenger: __C.FlutterBinaryMessenger) -> __C.FlutterMethodChannel in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          @objc static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.register(with: __C.FlutterPluginRegistrar) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          webview_cookie_manager.SwiftWebviewCookieManagerPlugin.handle(_: __C.FlutterMethodCall, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          reabstraction thunk helper from @escaping @callee_unowned @convention(block) (@unowned Swift.AnyObject?) -> () to @escaping @callee_guaranteed (@in_guaranteed Any?) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.(_setCookie in _E35A6B25B6E5F14C2C28AC213C9247D2)(cookie: __C.NSDictionary, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          closure #1 ([__C.NSHTTPCookie]) -> () in static webview_cookie_manager.SwiftWebviewCookieManagerPlugin.getCookies(url: Swift.String?, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
          ...
      "static Swift.String.== infix(Swift.String, Swift.String) -> Swift.Bool", referenced from:
          webview_cookie_manager.SwiftWebviewCookieManagerPlugin.handle(_: __C.FlutterMethodCall, result: (Any?) -> ()) -> () in libwebview_cookie_manager.a(SwiftWebviewCookieManagerPlugin.o)
    ld: symbol(s) not found for architecture x86_64
    clang: error: linker command failed with exit code 1 (use -v to see invocation)
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
    warning: Capabilities for Signing & Capabilities may not function correctly because its entitlements use a placeholder team ID. To resolve this, select a development team in the Runner editor. (in target 'Runner' from project 'Runner')

--- a/README.md
+++ b/README.md
@@ -34,6 +34,16 @@ await cookieManager.clearCookies();
  ```
 
  For more examples check example folder.
+ 
+ ## Troubleshooting
+ If you are using Objective C, check that PodFile have a flag use_frameworks ([why you need to do it](https://github.com/amag2511/webview_cookie_manager/issues/4#issuecomment-665508540))
+ ```
+target 'Runner' do
+  use_frameworks!
+  use_modular_headers!
+  ..........
+end
+```
 
 ## How it works
 The cookies stores and retrieves using the [httpCookieStore](https://developer.apple.com/documentation/webkit/wkwebsitedatastore/2881956-httpcookiestore) for iOS and [CookieManager](https://developer.android.com/reference/java/net/CookieManager) for Android.

